news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.png)
+
+I like maps. You like maps. We all like maps.
+
+For this reason, the mad scientists over at osu! labs have been synthesizing new ways to pump high-quality beatmaps directly into your bloodstream, and their latest creations are sure to excite you!
+
+Introducing three new categories of mappack!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/new-beatmap-pack-categories.png)
+
+## Project Loved packs
+
+Project Loved has been bringing some of osu!'s finest creations to your screens since 2016, and the time has come to bring some *HIGH DENSITY LOVED GOODNESS* to your songs folder. Each pack contains a selection of each season's greatest hits, as chosen by the Loved Captains themselves!
```

Comment:
No need for caps and I am not sure why this is italicized
```suggestion
Project Loved has been bringing some of osu!'s finest creations to your screens since 2016, and the time has come to bring some high density loved goodness to your songs folder. Each pack contains a selection of each season's greatest hits, as chosen by the Loved Captains themselves!
```

Reply:
gonna leave this unchanged / unresolved for now, as would like some discussion about allowing less formal newsposts with a bit more '''personality''' to them.

Reply:
this kind of loose tone stuff is fine, newsposts don't need to be ultra formal all of the time